nms/app/packages/magmalte/app/views/subscriber/SubscriberOverview.js

```diff
@@ -114,81 +108,56 @@ type SubscriberRowType = {
-  apnName: string,
-  sessionId: string,
-  ipAddr: string,
-  state: string,
-  activeDuration: string,
-  activePolicies: Array<string>,
-};
-
 type Props = {
   open: boolean,
   onClose?: () => void,
   imsi: string,
 };
 
-function JsonDialog(props: Props) {
-  const ctx = useContext(SubscriberContext);
-  const sessionState = ctx.sessionState[props.imsi] || {};
-  const configuredSubscriberState = ctx.state[props.imsi];
-  const subscriber: mutable_subscriber = {
-    ...configuredSubscriberState,
-    state: sessionState,
-  };
-  return (
-    <Dialog open={props.open} onClose={props.onClose} fullWidth={true}>
-      <DialogTitle>{props.imsi}</DialogTitle>
-      <DialogContent>
-        <ReactJson
-          src={subscriber}
-          enableClipboard={false}
-          displayDataTypes={false}
-        />
-      </DialogContent>
-    </Dialog>
-  );
-}
+export type TableProps = {
+  columns: Array<ActionTableColumn>,
+};
+const SubscriberTabs = withStyles({
+  indicator: {
+    backgroundColor: colors.secondary.dodgerBlue,
+  },
+})(Tabs);
 
-function SubscriberInternal(props: WithAlert) {
-  const {history, match, relativeUrl} = useRouter();
-  const [currRow, setCurrRow] = useState<SubscriberRowType>({});
-  const networkId: string = nullthrows(match.params.networkId);
-  const enqueueSnackbar = useEnqueueSnackbar();
-  const ctx = useContext(SubscriberContext);
-  const [refresh, setRefresh] = useState(true);
-  const classes = useStyles();
-  const [lastRefreshTime, setLastRefreshTime] = useState(
-    new Date().toLocaleString(),
-  );
+const SubscriberTab = withStyles({
+  root: {
+    fontFamily: typography.body1.fontFamily,
+    fontWeight: typography.body1.fontWeight,
+    fontSize: typography.body1.fontSize,
+    lineHeight: typography.body1.lineHeight,
+    letterSpacing: typography.body1.letterSpacing,
+    color: colors.primary.brightGray,
+    textTransform: 'none',
+  },
+})(Tab);
 
-  // Auto refresh subscribers every 30 seconds
-  const state = useRefreshingContext({
-    context: SubscriberContext,
-    networkId,
-    type: 'subscriber',
-    interval: REFRESH_INTERVAL,
-    enqueueSnackbar,
-    refresh,
-    lastRefreshTime,
-  });
+const Subscriber = withAlert(SubscriberTable);
+const SubscriberState = withAlert(SubscriberStateTable);
 
+export function SubscribersOverview() {
+  const {history, relativeUrl} = useRouter();
+  const classes = useStyles();
+  const [currTabIndex, setCurrTabIndex] = useState<number>(0);
   const networkCtx = useContext(NetworkContext);
-  // $FlowIgnore
-  const subscriberMap: {[string]: subscriber} = state.state;
-  // $FlowIgnore
-  const sessionState: {[string]: subscriber_state} = state.sessionState;
-  const subscriberMetrics = ctx.metrics;
-  const [jsonDialog, setJsonDialog] = useState(false);
+  const ctx = useContext(SubscriberContext);
+  const subscriberMap = ctx.state;
+  const subscribersTabList: Array<string> = ['Susbcribers', 'Subscriber State'];
```

Comment:
Thinking about this further, These should be named as 
"subscribers" -> "Config" 
"subscriber state" -> "Sessions"

If it isn't too much work, It might be nicer if we can perhaps add tabs to TopBar itself I think. Sorry for suggesting this to be similar to existing policy tabs. I was thinking this way because there is quite a bit of real estate there and we can add sessions and other subscriber bulk related views there.

Reply:
Oh ok you're right. This should not be too much work !